packages/cli/src/commands/generate/forms/generate_forms_command.ts

```diff
@@ -102,6 +104,35 @@ export class GenerateFormsCommand
           error
         );
       }
+      if (
+        error instanceof BackendOutputClientError &&
+        error.code === BackendOutputClientErrorType.EXPIRED_TOKEN
```

Comment:
nit: micro-optimization thoughts...

```suggestion
        if(error instanceof BackendOutputClientError){
           switch(error.code){
            ...
```